<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1211</width>
     <height>761</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelphoto">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>151</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:25px</string>
    </property>
    <property name="text">
     <string>数据管理系统</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonInput">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>输入数据</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonDelete">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_10">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>导入</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_11">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>备份</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonExport">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>导出</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonexit">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonPrinter">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>打印</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>851</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>281</width>
      <height>611</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>证件号码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>180</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>220</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>性别：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>260</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>出生年月：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>地址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelID">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>140</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelname">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>180</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelgender">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>220</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelbirth">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>260</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labeladdress">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>300</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
